v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FUtils.scala

```diff
@@ -65,6 +68,37 @@ object V2FUtils {
       }
     }
 
+  /**
+    * Given the SCollection of ReadableFiles that contains TSVs convert each TSV to a Msg and get its filepath.
+    *
+    * @param tableName the name of the TSV table that was converted to Msg
+    */
+  def tsvToMsg(
+    tableName: String
+  ): SCollection[ReadableFile] => SCollection[(String, Msg)] =
+    _.transform(s"Convert $tableName TSVs to Msg") { collection =>
```

Comment:
The string passed to `transform` ends up being shown in Dataflow's UI as the description of the step, so I wonder if `Convert <table> TSVs to Msg` is too implementation-specific. Would suggest a change:
```suggestion
    _.transform(s"Extract $tableName TSV rows") { collection =>
```